cylc/flow/task_pool.py

```diff
@@ -1285,24 +1329,30 @@ def force_trigger_tasks(self, items, reflow=False):
         n_warnings, task_items = self.match_taskdefs(items)
         flow_label = self.flow_label_mgr.get_new_label()
         for name, point in task_items.keys():
-            # Already in pool? Keep merge flow labels.
-            itask = self.get_task(name, point, flow_label)
-            if itask is None:
-                # Spawn with new flow label.
-                itask = self.spawn_task(name, point, flow_label, reflow=reflow)
+            # Already in pool? Keep it and merge flow labels.
+            itask = self.get_task_main(name, point, flow_label)
             if itask is not None:
+                # Trigger existing task proxy
+                LOG.critical('setting %s ready to run', itask)
```

Comment:
Why is this `critical`? Leftover debugging?

Reply:
No, this has been (wrongly) `critical` for ages. It's not really a change on this branch, just part of a cut-and-paste move of several lines to another location.

However, I actually changed it to `info` myself in f01c4233c0 from #4218 so I'm a bit confused as to why it's shown up as `critical` again here. Anyhow, I'll (re)fix it.